src/Contracts/HasApiTokens.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laravel\Sanctum\Contracts;
+
+interface HasApiTokens
+{
+    /**
+     * Get the access tokens that belong to model.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\MorphMany
+     */
+    public function tokens();
+
+    /**
+     * Determine if the current API token has a given scope.
+     *
+     * @param  string  $ability
+     * @return bool
+     */
+    public function tokenCan(string $ability);
+
+    /**
+     * Create a new personal access token for the user.
+     *
+     * @param  string  $name
+     * @param  array  $abilities
+     * @return \Laravel\Sanctum\NewAccessToken
+     */
+    public function createToken(string $name, array $abilities = ['*']);
+
+    /**
+     * Get the access token currently associated with the user.
+     *
+     * @return \Laravel\Sanctum\Contracts\HasAbilities
+     */
+    public function currentAccessToken();
+
+    /**
+     * Set the current access token for the user.
+     *
+     * @param  \Laravel\Sanctum\Contracts\HasAbilities  $accessToken
+     * @return HasApiTokens
```

Comment:
This can't be a trait. Trait isn't a valid type to return in DocBlocks as they're not objects or interfaces.

Reply:
Sorry, realized this is an interface of course. Needs to be an FQN:

```suggestion
     * @return \Laravel\Sanctum\Contracts\HasApiTokens
```

Reply:
It doesn't return a trait. That is the contract (itself) with the same name.